sdk/managed/Microsoft.WindowsAzure.Mobile.Managed_IncludeXamarin.sln

```diff
@@ -117,9 +117,9 @@ Global
 		{A2500BD2-A9A2-4085-9978-DEE3675B02DD}.Signed|Mixed Platforms.Build.0 = Release|Any CPU
 		{A2500BD2-A9A2-4085-9978-DEE3675B02DD}.Signed|x64.ActiveCfg = Release|Any CPU
 		{A2500BD2-A9A2-4085-9978-DEE3675B02DD}.Signed|x86.ActiveCfg = Release|Any CPU
-		{4864F1AA-A4B9-4F32-B319-A9BAC7D1B200}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{4864F1AA-A4B9-4F32-B319-A9BAC7D1B200}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{4864F1AA-A4B9-4F32-B319-A9BAC7D1B200}.Debug|Any CPU.Deploy.0 = Debug|Any CPU
+		{4864F1AA-A4B9-4F32-B319-A9BAC7D1B200}.Debug|Any CPU.ActiveCfg = Debug|x64

```

Comment:
PR #99 breaks Microsoft.WindowsAzure.Mobile.Managed_IncludeXamarin.sln build, so this PR is to fix the broken build. Still more work need to be done to make Xamarin e2e tests uploading to sunlight.